src/stories/packages/components/ExampleLink.stories.ts

```diff
@@ -0,0 +1,70 @@
+import type { Meta, StoryObj } from '@storybook/react';
+import { Link } from '@terra-ui-packages/components';
+
+const meta: Meta<typeof Link> = {
+  title: 'Packages/Components/Link',
+  component: Link,
+  parameters: {
+    layout: 'centered',
+  },
+  tags: ['autodocs'],
+  argTypes: {
+    children: {
+      control: 'text',
+      description: 'Link text',
+    },
+    href: {
+      control: 'text',
+      description: 'URL destination of the link',
+    },
+    tooltip: {
+      control: 'text',
+      description: 'Tooltip text',
+    },
+    tooltipSide: {
+      control: 'select',
+      options: ['top', 'right', 'bottom', 'left'],
+      description: 'Tooltip position',
+      table: {
+        defaultValue: { summary: 'bottom' },
+      },
+    },
+    tooltipDelay: {
+      control: 'number',
+      description: 'Tooltip delay in milliseconds',
+      table: {
+        defaultValue: { summary: 'undefined' },
+      },
+    },
+    disabled: {
+      control: 'boolean',
+      description: 'Should the link be disabled?',
+      table: {
+        defaultValue: { summary: 'false' },
+      },
+    },
+    variant: {
+      control: 'select',
+      options: [undefined, 'light'],
+      description: 'Default color, or lighter variant?',
+      table: {
+        defaultValue: { summary: 'undefined' },
+      },
+    },
```

Comment:
Since `baseColor` is part of our public interface for this component, I do think we should document it. You can add this, which doesn't allow folks to try it out, but does allow them to see that it exists.

```
baseColor: {
      description: 'function to return the color of the link, takes an numeric intensity value as an argument',
      type: 'function'
    }
```
![image](https://github.com/DataBiosphere/terra-ui/assets/484484/3b71f71a-f922-42e9-9bde-34298a7c7fbd)